code/modules/events/mass_hallucination.dm

```diff
@@ -6,6 +6,9 @@
 		var/mob/living/carbon/human/H = thing
 		if(H.stat == DEAD)
 			continue
+		var/turf/T = get_turf(H)
+		if(!is_station_level(T.z))
+			continue
```

Comment:
I assume this is done with way to account for lockers? Otherwise `if(!is_station_level(H.z))` would work just fine here, without needing another var declaration

Reply:
Tested this just in case (wasn't 100% sure myself). Lockers, mechs etc do put you on z level 0.

Reply:
Not just lockers. Sleepers, cryotubes, mechs, machines of all kinds, etc.